lib/internal/per_context/primordials.js

```diff
@@ -266,4 +266,4 @@ primordials.SafeStringIterator = createSafeIterator(
 );
 
 Object.setPrototypeOf(primordials, null);
-Object.freeze(primordials);
+Object.seal(primordials);
```

Comment:
Wouldn't it be better to only do this if the `--expose-primordials` option is passed in?

Reply:
It would, but I'm not sure how to get the information here (`require` is not yet defined when this line is executed).